Modules/vc-module-catalog/VirtoCommerce.CatalogModule.Web/Scripts/selectors/catalog-selector.tpl.html

```diff
@@ -0,0 +1,8 @@
+<div class="form-input" ng-controller="virtoCommerce.catalogModule.catalogSelectorController">
+    <ui-select multiple ng-model="blade.currentEntity.catalogIds">
+        <ui-select-match placeholder="{{'catalog.selectors.placeholders.catalog-lists' | translate}}">{{$item.name}}</ui-select-match>
+        <ui-select-choices repeat="x.id as x in catalogs | filter: { name: $select.search }">
+            <span ng-bind-html="x.name | highlight: $select.search"></span>
```

Comment:
I think better to move this selector to pricing module for now (though it is catalog selector), and revert all changes in pricing one. Do not know if if would be shared across modules.